test/unit/sidekiq_logging_middleware_test.rb

```diff
@@ -7,13 +9,13 @@ class SidekiqLoggingMiddlewareTest < ActiveSupport::TestCase
       'jid' => 123,
       'args' => [
         {
-          "some_arg": "value",
```

Comment:
This is just because we are not supposed to have hashes with symbols in Sidekiq job argruments.